package.json

```diff
@@ -126,10 +126,11 @@
     "dev:https": "HTTPS=true node webpack_config/server.js",
     "predev:https": "check-node-version --package",
     "derivation-checker": "webpack --config=./webpack_config/webpack.derivation-checker.js && node ./dist/derivation-checker.js",
-    "tslint": "tslint --project . --exclude common/vendor/**/*",
+    "tslint": "tsc --noEmit && tslint --project . --exclude common/vendor/**/*",
```

Comment:
I think this should be a semicolon instead of a `&&` that way we get both tsc errors and tslint errors, if tsc fails

Reply:
The problem with that is that it would only fail on the right-side command. If the first one fails, and the second one passes, I don't think the build would fail. Maybe I should make these 2 separate commands.